docs/tutorials/writing-sd-card-image-etcher.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Writing an SD Card Image Using Etcher (Cross-Platform)
+subject: SD Card Image
+---
+
+1. Download and install Etcher from [their website](http://etcher.io).
+2. Launch Etcher after it has finished installing.
+
+   <div>{% include screenshot.html source="/images/etcher/home.png" caption="Etcher home screen" %}</div>
```

Comment:
what does it look like without the `<div>` here? The rest of the tutorial pages have images like this without the `<div>` around them.